addons/spark/0.x/spark.yaml

```
@@ -0,0 +1,31 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: spark
+  labels:
+    kubeaddons.mesosphere.io/name: spark
+    kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
+  annotations:
+    stage.kubeaddons.mesosphere.io/spark: Beta
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.1.0-2"
+    appversion.kubeaddons.mesosphere.io/spark: "0.2.0"
+spec:
+  kubernetes:
+    minSupportedVersion: v1.16.0
+  cloudProvider:
+    - name: aws
+      enabled: true
+    - name: docker
+      enabled: true
+    - name: none
+      enabled: true
+  requires:
+    - matchLabels:
+        kubeaddons.mesosphere.io/name: prometheus
```

Why does spark require prometheus? Because of CRDs or some tight integration with Prometheus?

---

Yes, it requires `ServiceMonitor` CRD to be present

---

Ok :+1: maybe we can just add a quick comment?
```suggestion
        # requires ServiceMonitor CRD to be present
        kubeaddons.mesosphere.io/name: prometheus
```